q2_cutadapt/tests/test_demux.py

```diff
@@ -519,35 +519,24 @@ def test_di_duplicate_barcode_pairs(self):
                                  forward_barcodes=forward_barcodes,
                                  reverse_barcodes=reverse_barcodes)
 
-    def test_multiple_orientations_dual_indices(self):
+    def test_di_duplicate_barcode_pairs_mixed(self):
```

Comment:
```suggestion
    def test_dual_index_duplicate_barcode_pairs_mixed(self):
```
Same as other test comment